styleguide/_themes/derek/scss/globalElements/basement.scss

```diff
@@ -25,61 +30,78 @@
   width: 20px;
 }
 
-@include basement-flag(argentina, 0, -280px);
+.basement-copyright {
+  @include make-sm-column(3);
+  @include make-sm-column-offset(6);
+}
+
+.basement-copyright-text {
+  color: $medium-gray;
```

Comment:
`$medium-gray` is no longer a valid color.  Consult the variables here: https://github.com/rackerlabs/zoolander/blob/master/styleguide/_themes/global/scss/swatch.scss